src/main/java/core/basesyntax/FigurePrinter.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigurePrinter {
+    String draw();
```

Comment:
```suggestion
public interface InfoDrawer {
    void draw();
```

lets do not use word Figure in interface name, as we could use this interface not only for figure classes. And draw method should be void